README.md

```diff
@@ -100,6 +100,21 @@ _Note: SwiftGen needs Xcode 8.3 to build, so installing via Homebrew requires yo
 
 ---
 </details>
+<details>
+<summary>Via <strong>Mint</strong></summary>
```

Comment:
@yonaskolb What's the official name/wording of your tool? Is it `Mint`, `mint`, or `Mint 🌱`? My OCD wants it to be mentioned properly 😛

Reply:
Good question. The repo is called `Mint` and the executable is `mint` (Much like `SwiftGen`).
The `🌱` is an optional adornment 😛